pkg/printer/printer.go

```diff
@@ -107,17 +109,12 @@ func (p *Printer) printK8sValidationWarning(warning Warning) {
 	fmt.Fprintln(out)
 }
 
-func (p *Printer) PrintYamlSchemaResults(result *JSONSchemaValidatorResults, error error) {
-	if result == nil {
-		p.printInColor("INVALID FILE PATH\n", p.Theme.Colors.RedBold)
-		return
-	}
-	if result.Errors() != nil {
+func (p *Printer) PrintYamlSchemaResults(errorsResult []jsonschema.Detailed, error error) {
+	if errorsResult != nil {
```

Comment:
It took me a while to realize there is an early return in this if, maybe changing to `if{}...else{}` will be more readable